modules/__tests__/prefixer-test.js

```diff
@@ -1,119 +1,113 @@
+  var exenv;
+
   beforeEach(() => {
     browserPrefix = '';
     mockStyle = {};
 
-    document.createElement = jest.genMockFunction().mockImplementation(() => {
+    sinon.stub(document, 'createElement', () => {
       return {style: mockStyle};
     });
-    window.getComputedStyle = jest.genMockFunction().mockImplementation(() => {
+    sinon.stub(window, 'getComputedStyle', () => {
       return [browserPrefix + 'transform'];
     });
 
-    jest.setMock('exenv', {
+    exenv = {
       canUseDOM: true,
       canUseEventListeners: true
-    });
+    };
   });
 
   afterEach(() => {
-    document.createElement = originalDocumentCreateElement;
-    window.getComputedStyle = originalWindowGetComputedStyle;
+    document.createElement.restore();
+    window.getComputedStyle.restore();
   });
 
   it('detects Firefox prefix', () => {
     browserPrefix = '-moz-';
-    var Prefixer = require('../prefixer.js');
-    expect(Prefixer.cssPrefix).toBe('-moz-');
-    expect(Prefixer.jsPrefix).toBe('Moz');
+    var Prefixer = require('inject!prefixer.js')({'exenv': exenv});
+    expect(Prefixer.cssPrefix).to.equal('-moz-');
+    expect(Prefixer.jsPrefix).to.equal('Moz');
   });
 
   it('detects IE prefix', () => {
     browserPrefix = '-ms-';
-    var Prefixer = require('../prefixer.js');
-    expect(Prefixer.cssPrefix).toBe('-ms-');
-    expect(Prefixer.jsPrefix).toBe('ms');
+    var Prefixer = require('inject!prefixer.js')({'exenv': exenv});
+    expect(Prefixer.cssPrefix).to.equal('-ms-');
+    expect(Prefixer.jsPrefix).to.equal('ms');
   });
 
   it('detects Opera prefix', () => {
     browserPrefix = '-o-';
-    var Prefixer = require('../prefixer.js');
-    expect(Prefixer.cssPrefix).toBe('-o-');
-    expect(Prefixer.jsPrefix).toBe('O');
+    var Prefixer = require('inject!prefixer.js')({'exenv': exenv});
+    expect(Prefixer.cssPrefix).to.equal('-o-');
+    expect(Prefixer.jsPrefix).to.equal('O');
   });
 
   it('detects Webkit prefix', () => {
     browserPrefix = '-webkit-';
-    var Prefixer = require('../prefixer.js');
-    expect(Prefixer.cssPrefix).toBe('-webkit-');
-    expect(Prefixer.jsPrefix).toBe('Webkit');
+    var Prefixer = require('inject!prefixer.js')({'exenv': exenv});
+    expect(Prefixer.cssPrefix).to.equal('-webkit-');
+    expect(Prefixer.jsPrefix).to.equal('Webkit');
   });
 
   it('doesn\'t detect prefix if in server', () => {
-    jest.setMock('exenv', {
+    exenv = {
       canUseDOM: false,
       canUseEventListeners: false
-    });
+    };
 
-    var Prefixer = require('../prefixer.js');
+    var Prefixer = require('inject!prefixer.js')({'exenv': exenv});
     var result = Prefixer.getPrefixedStyle({display: 'flex'});
 
-    expect(result).toEqual({display: 'flex'});
-    expect(Prefixer.cssPrefix).toBe('');
-    expect(Prefixer.jsPrefix).toBe('');
-    expect(document.createElement).not.toBeCalled();
-    expect(window.getComputedStyle).not.toBeCalled();
+    expect(result).to.eql({display: 'flex'});
```

Comment:
@ryan-roemer is right, `.equal` would fail for these objects. @ianobermiller, are you saying you prefer the `.deep.equal` syntax to `.eql`?